File: java/test/_polytrack5.java
import liba.polytrack5;

class _polytrack5 {
    public static void main(String[] args) {
        polytrack5 obj = new polytrack5(0, 1, 0, 1, 0, 1, 0, 1);
        double[] out = obj.out(0.5);
        System.out.print(obj.pos(0.5) + "," + obj.vec(0.5) + "," + obj.acc(0.5));
        System.out.println(" " + out[0] + "," + out[1] + "," + out[2]);
    }
}

File: quickjs/test/polytrack5.js
import * as a from "liba.so";
var polytrack5 = new a.polytrack5(0, 1, 0, 1)
polytrack5.gen(0, 10, 0, 10, 0, 0)
for (var i = 0; i < 10; i += 0.1) {
    console.log(i, polytrack5.pos(i), polytrack5.vel(i), polytrack5.acc(i))
}
